roles/ovirt-image-template/README.md

```diff
@@ -1,7 +1,9 @@
 oVirt Image Template
 ====================
 
-The `ovirt-image-template` role downloads a QCOW2 image from a specified URL and creates a template from it.
```

Comment:
I would rephrase it to:

The `ovirt-image-template` role creates a template from external image. Currently the disk can be a image in Glance external provider or QCOW2 image.